### PR TITLE
Repeat yelling option

### LINKED ISSUE #8: [Feature Request] Play sounds more than once per move
Add a setting to allow the user to choose whether a sound should play again if they have not moved after the first sound. The extra sounds could pull from the same pool, or also have a separate pool for "especially long turns".

### PR DESCRIPTION
**Pull request by rootEnginear**
Hi! This PR implements the "repeat yelling" option into the extension (close #8). The logic behind it is very straightforward by creating an interval after the first sound has been played. As always, if you found any flaws in my code, please tell me 😉.

### CODE REVIEW

File: js/chessMoveReminder.js [hunk 1]
```diff
@@ -94,7 +91,22 @@ function chessMoveReminder() {
...
-                            timer = setTimeout(playAudio, timeToWait, who);
+                            timer = setTimeout(playAudio, timeToWait, formattedWho);
+                            
+                            if (repeatEnabled) {
+                                setTimeout(function(){
+                                    if (repeatType === 'percentage') {
+                                        let seconds =
+                                        parseSecondsFromClock(currentClock) *
+                                        repeatNumber *
+                                        10;
```
Comment (shuhaoli) [inline js/chessMoveReminder.js:102]: Let's refactor this repeated code into a function

File: js/chessMoveReminder.js [hunk 2]
```diff
@@ -94,7 +91,22 @@ function chessMoveReminder() {
...
+                                        timeToWait = Math.max(seconds, 2000);
+                                    } else {
+                                        timeToWait = repeatNumber * 1000;
```
Comment (shuhaoli) [inline js/chessMoveReminder.js:105]: Try out 0 second repeat :D

We should probably set a floor for this repeat (what's the longest sound clip that we have?)

**Review by shuhaoli — COMMENTED** [review]
Thanks for the work!

**Comment by rootEnginear** [comment]
Sorry for the late fix due to my examination week. I have adjusted the allowed repeating interval to at least 5 seconds (because the longest one is about 5 secs). As usual, if you have encountered any bugs, please tell me.

**Comment by shuhaoli** [comment]
Sorry for the very late response. I kept getting more and more voices yelling at me for some reason and I couldn't figure out why, but I know how to fix it now. I will merge in your PR, thanks for all the work!

**Pull request merged**